src/components/forms/groups/Create.js

```diff
@@ -52,6 +52,25 @@ class CreateGroupForm extends Component {
             setInitialFocus
           />
         </div>
+        <div>
+          <div>
+            <label>Día de entrega</label>
+          </div>
+          <Field
+            name="delivery"
+            component="select"
+            placeholder="Especifica el dia de entrega de la compra"
+          >
+            <option></option>
+            <option value="monday">Lunes</option>
+            <option value="tuesday">Martes</option>
+            <option value="wednesday">Miércoles</option>
+            <option value="thursday">Jueves</option>
+            <option value="friday">Viernes</option>
+            <option value="saturday">Sábado</option>
+            <option value="sunday">Domingo</option>
+          </Field>
```

Comment:
The `Input` component can render a `select` too... https://github.com/coopdevs/katuma-web/blob/develop/src/components/Input/index.js#L66

In any case, yes maybe we can create a `WeekDaysSelect` and reuse it for both the `delivery` and `confirmation`.